uvm/js/common/util/Util.js

```diff
@@ -649,6 +649,34 @@ Ext.define('Ung.util.Util', {
         return true;
     },
 
+    validateFields: function (form) {
```

Comment:
This seems too generic for a use that is only in the device controller.  I recommend moving it there instead of this file.

Reply:
Moved this method to DevicesController